pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -613,13 +616,17 @@ func (ctrl *Controller) syncImageConfig(key string) error {
 				if err != nil {
 					return fmt.Errorf("could not update registries config with new changes: %v", err)
 				}
+				policyJSON, err = updatePolicyJSON(dataURL.Data, blockedRegs, allowedRegs)
```

Comment:
This is passing the `originalRegistriesIgn` (for `registries.conf`) data contents to `updatePolicyJSON` (for `policy.json`); luckily that does not break because `updatePolicyJSON` completely ignores the input — but that also means that the original template is completely ignored by `updatePolicyJSON` (and if I understand the render controller correctly, that means that the original template is completely ignored _if_ `insecureRegs != nil || blockedRegs != nil || allowedRegs != nil` and used unmodified otherwise).

Reply:
Fixed as well.